doc/reference.md

```diff
@@ -31,6 +31,7 @@ A local file `config.sh` is required for Bento, it contains variable used to gen
 
 - `CHROOT_DIR`: the directory in which the SFTP server will be configured to serve files from
 - `REMOTE_IP`: the IP address or hostname used by SFTP client to reach the server with the configuration files
+- `REMOTE_PORT`: the port of the IP address or hostname used by SFTP client to reach the server with the configuration files. Defaults to 22.
```

Comment:
actually, it doesn't default to 22, you just placed 22 in the config sample, if I try using the new version without updating my `config.sh`, I'll have errors.

Could you add a check to stop bento if `REMOTE_PORT` isn't set?

Reply:
Ah, you're right! 
I changed `REMOTE_PORT` to be set by default in  `bento`, so it should indeed now be optional.

The default port by ssh (and sftp) is 22 so I think its safe to set like that.